include/taskolib/Step.h

```diff
@@ -76,7 +76,7 @@ class Step
      * This function performs the following steps:
      * 1. A fresh script runtime environment is prepared and safe library components are
      *    loaded into it.
-     * 2. The lua_init_function from the context is run if it is defined (non-null).
+     * 2. The lua_step_setup from the context is run if it is defined (non-null).
      * 3. Selected variables are imported from the context into the runtime environment.
```

Comment:
Actually there is a new item here: 2a. The step setup script given as a parameter is run

Reply:
Why is this 2a and 2b and not 2 and 3?

Reply:
Sorry, I did not mean that literally. Of course 2 and 3 is preferable over 2a and 2b.

Reply:
Fixed.